evol/logger.py

```diff
@@ -3,27 +3,32 @@
 import json
 import logging
 import sys
+import uuid
 
 class BaseLogger():
     """
-    The `evol.BaseLogger` is the most basic logger in evol. You can add it to a population
-    so that the population knows how to handle the `.log()` verb. 
+    The `evol.BaseLogger` is the most basic logger in evol. 
+    You can supply it to a population so that the population 
+    knows how to handle the `.log()` verb. 
     """
-    def __init__(self, file='/tmp/evol-logs.log', stdout=False, format='%(asctime)s,%(message)s'):
+    def __init__(self, file=None, stdout=False, format='%(asctime)s,%(message)s'):
         self.file = file
         if file is not None:
             if not os.path.exists(os.path.split(file)[0]):
                 raise RuntimeError(f"path to file {os.path.split(file)[0]} does not exist!")
```

Comment:
This should be a more specific error, I think `FileExistsError` exists.

Reply:
I currently give an error if the filepath to the file **does not** exist. it is fine to append to a file that already exists, although a warning may be in order.

Reply:
Sorry, I meant `FileNotFoundError`